sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -26,6 +26,10 @@ public partial class TextAnalyticsClient
         private readonly string DefaultCognitiveScope = "https://cognitiveservices.azure.com/.default";
         private const string AuthorizationHeader = "Ocp-Apim-Subscription-Key";
 
+        // Specifies the method used to interpret string offsets. Default to <see cref="StringIndexType.Utf16CodeUnit"/>.
+        private readonly StringIndexType _stringCodeUnit = StringIndexType.Utf16CodeUnit;
```

Comment:
Just out of curiosity, do we intend to let users overwrite this value?

Reply:
Not currently. There was some talking about the service returning 2 Offset and Length with different encoding and us exposing them, but for now only the one the language allows (UTF-16 for C#).